memory_w25q1128jv_spi/memory_w25q1128jv_spi.h

```diff
@@ -0,0 +1,186 @@
+/*
+ * W25Q128JV_MEMORY.h
+ *
+ *  Created on: Jan 1, 2024
+ *      Author: rober
+ */
+#pragma once
+
+#include <cmath>
+
+#if defined(STM32F1)
+#include "stm32f1xx_hal.h"
+#endif
+
+#ifndef SPI_Timeout
+#define SPI_Timeout 10
+#endif
+
+class MemoryW25q1128jvSpi {
+  public:
+    enum class State { CHIP_ENABLE, PAGE_WRITE, COMPLETE, ERROR };
+
+    /**
+     * @param hspi 		 SPI bus handler
```

Comment:
why is this line off and why did the formatter not fix it lol

Reply:
rip, will fix